source/photo_audio_video_file.md

```diff
@@ -0,0 +1,148 @@
+device camera or attach a video file. This may be particularly useful when a
+question requires the respondent to demonstrate something.
+
+If a question requires that you attach a file such as a PDF, you can use the
+“File” question type.
+
+Note: Your device should also have an in-built camera and audio recorder to use
+the "photo", "audio" and "video" question types while collecting data..
+
+## How to set up "photo", "audio", "video" and "file" question types
+
+### Setting up in FormBuilder
+
+Adding media questions on the form is easy and does not require any extra setup:
+
+-   In the KoboToolbox Formbuilder, click the + button to add a new question
+-   Type the question text, for example “Take a picture of the housing unit”,
+    then click ADD QUESTION or press ENTER on your keyboard
+-   Choose the question type
+
+![Adding media question](images/photo_audio_video_file/add.gif)
+
+### Setting up in XLSForm
+
+To add media questions in xlsform, use the `image`, `audio`, `video`, and `file`
+question types as shown in the following example:
+
+| type  | name        | label                                                       | hint            |
+| :---- | :---------- | :---------------------------------------------------------- | :-------------- |
+| image | house_photo | Take a photo of the housing unit                            |                 |
+| audio | impact      | What has been the impact of the project on your household?  | Record as audio |
+| video | preparation | Record video of the respondent as they prepare the VitaMeal |                 |
+| file  | CV          | Attach your CV                                              |                 |
+
+## Appearance of Photo, audio, video and file question types in web forms and KoboCollect
+
+### Default appearance
+
+![Default appearances](images/photo_audio_video_file/default_appearances.png)
+
+### Advanced appearances for "photo" question type
+
+When adding the “Photo” question type, you can choose from a number of
+appearances (under the question settings). Appearances change the way the
+question is displayed on web forms or on KoboCollect.
+
+![Advanced appearances for photo question type](images/photo_audio_video_file/advanced_appearances_photo.png)
+
+![Advanced appearances](images/photo_audio_video_file/advanced_appearances.png)
+
+### Adding advanced appearances in xlsform
+
+You can specify advanced appearances of the "photo" question in XLSForm under
+the appearance column as shown in the following example:
+
+| type  | name       | label                      | appearance |
+| :---- | :--------- | :------------------------- | :--------- |
+| image | sign       | Sign here                  | signature  |
+| image | drawing    | Draw here                  | draw       |
+| image | annotation | Take an image and annotate | annotate   |
+
+## Background audio recording
+
+KoboToolbox now allows you to record audio in the background when you open the
```

Comment:
I think we should avoid writing in this style as it implies that it's new — and how long is something considered new for? It's also technically incorrect since it's Collect that does the background recording, not Kobo — we just facilitate them setting the option in the formbuilder.
> KoboToolbox now allows you

Reply:
Corrected